test/unit/mdc-ripple/foundation-deactivation.test.js

```diff
@@ -101,6 +101,43 @@ testFoundation('runs deactivation UX on public deactivate() call', ({foundation,
   td.verify(adapter.addClass(cssClasses.FG_BOUNDED_ACTIVE_FILL));
 });
 
+testFoundation('runs deactivation UX mousedown pressed for longer than deactivation timeout',
+      ({foundation, adapter, mockRaf}) => {
+  const handlers = captureHandlers(adapter);
+  foundation.init();
+  mockRaf.flush();
+
+  handlers.mousedown();
+  mockRaf.flush();
+
+  setTimeout(() => {
+    td.verify(adapter.removeClass(cssClasses.BG_ACTIVE));
+    td.verify(adapter.addClass(cssClasses.BG_BOUNDED_ACTIVE_FILL));
+    td.verify(adapter.addClass(cssClasses.FG_BOUNDED_ACTIVE_FILL));
+  }, foundation.DEACTIVATION_TIMEOUT_MS);
+});
+
+testFoundation('does not run deactivation UX on mouseup after mousedown pressed for longer than deactivation timeout',
+      ({foundation, adapter, mockRaf}) => {
+  const handlers = captureHandlers(adapter);
+  foundation.init();
+  mockRaf.flush();
+
+  handlers.mousedown();
+  mockRaf.flush();
+
+  setTimeout(() => {
+    td.verify(adapter.removeClass(cssClasses.BG_ACTIVE));
```

Comment:
Not exactly sure how these tests are different from the ones above...

Should this be verifying that these adapter methods are _not_ called _before_ `DEACTIVATION_TIMEOUT_MS`?